django/contrib/auth/admin.py

```diff
@@ -80,9 +82,11 @@ def get_form(self, request, obj=None, **kwargs):
         return super(UserAdmin, self).get_form(request, obj, **defaults)
 
     def get_urls(self):
-        return [
-            url(r'^(.+)/password/$', self.admin_site.admin_view(self.user_change_password), name='auth_user_password_change'),
-        ] + super(UserAdmin, self).get_urls()
+        return [url(
```

Comment:
I agree about limiting line length, just not as part of this PR. I've reopened https://code.djangoproject.com/ticket/23395.